errors.js

```diff
@@ -2,4 +2,14 @@ const serverErrors=((err,req,res,next)=>{
     res.status(500).send({status:500,msg:"internal server error"})
 })
 
-module.exports = {serverErrors}
\ No newline at end of file
+const customErrors = ((err,req,res,next)=>{
+    if(err.status){
+        res.status(err.status).send({msg: err.msg})
+    }
+    if(err.code === '22P02'){
```

Comment:
Ideally, we'd like the PSQL error handler to be separate from the custom error handler